templates/launch.html

```diff
@@ -1,477 +1,477 @@
+
+        // store fetch so it only needs to be re-done if the survey changes
+        let supplementaryDataSets = null;
+
+        function clearSurveyMetadataFields() {
+            document.querySelector('#survey_metadata_fields').innerHTML = ""
+            hideSupplementaryData()
+        }
+
+        function showSupplementaryData () {
+            document.querySelectorAll(".supplementary-data--hidden").forEach(element => element.style.display = 'block');
+        }
+
+        function hideSupplementaryData () {
+            document.querySelectorAll(".supplementary-data--hidden").forEach(element => element.style.display = 'none');
+        }
+
+        function showAllDivs() {
+            document.querySelectorAll("#survey_metadata, .field-container--hidden, .group-title--hidden").forEach(function (element) {
+                element.style.display = 'block';
+            });
+
+            let launchPattern = document.querySelector("#launch_pattern").value
+
+            if (launchPattern === "v1") {
+                document.querySelectorAll(".Social").forEach(function(element) {element.setAttribute("disabled", "disabled")})
```

Comment:
this function has been removed from the `onchange` method of changing the launcher version, so I don't know if we'll get correct behaviour when switching from v2 to v1 (if we needed to) because this isn't going to happen

but at the same time I don't know when `document.querySelectorAll(".Social")` would return anything, it doesn't for social surveys... know when its used?

Reply:
I agree I cant see where that would be being used. There is no elements that I can see that have a `.social` class